helper/md-test.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+set -eu
+
+readonly IMAGES=( registry.gitlab.com/enarx/misc-testing/ubuntu-base:latest
+                  registry.gitlab.com/enarx/misc-testing/debian-base:latest
+                  registry.gitlab.com/enarx/misc-testing/centos7-base:latest
+                  registry.gitlab.com/enarx/misc-testing/centos8-base:latest
+                  registry.gitlab.com/enarx/misc-testing/fedora-base:latest )
+readonly CONTEXTS_NON_KVM=( git,helloworld crates,helloworld ) 
+readonly CONTEXTS_KVM=( git,helloworld,kvm-helloworld,kvm crates,helloworld,kvm-helloworld,kvm ) 
+# readonly CONTEXTS_SGX=( sgx,git,helloworld,sgx-helloworld sgx,crates,helloworld,sgx-helloworld ) 
+# readonly CONTEXTS_SNP=( snp,git,helloworld,kvm-helloworld,kvm snp,crates,helloworld,kvm-helloworld,kvm ) 
+readonly HOMEDIR="/home/user"
+
+test_image_kvm() {
+    local markdown_doc_path="$(realpath "$1")"
+    local image="$2"
+    local context="$3"
+    echo -e "\n\nRunning: ${image} (KVM)"
+    echo "Context: \"${context}\""
+    echo "Markdown Document: \"${markdown_doc_path}\""
+    docker run \
```

Comment:
why not use podman if you run on our self-runners anyway?

Reply:
We are not running on our self-hosted runners. KVM is for local testing. Goal is to transition to the direct/nil backend and remove the KVM backend completely.